creating-virtual-machines/dedicated-cpu.adoc

```diff
@@ -150,6 +150,8 @@ data:
 Sidecar containers and CPU allocation overhead
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+*Note:* In order to run sidecar containers, KubeVirt requires `Sidecar` feature gate to be enabled
```

Comment:
Looks good. It might be worth adding a little more clarification here. In the context of the whole doc, i don't think what you have is confusing, but in isolation it might be helpful if it read more like the following...

"In order to run sidecar containers, KubeVirt requires the `Sidecar` feature gate to be enabled by adding `Sidecar` to the kubevirt-config ConfigMap's `feature-gates` field."

Reply:
Yes, it sounds better. Updated.